setup.py

```diff
@@ -87,31 +87,33 @@ def get_version(version_file):
     package_data={
         "pe_reports": ["data/shell/*.pptx", "data/*.ini"],
         "pe_mailer": ["data/*"],
+        "pe_source": ["data/*"],
```

Comment:
Keep this in alphabetical order:
```suggestion
        "pe_mailer": ["data/*"],
        "pe_reports": ["data/shell/*.pptx", "data/*.ini"],
        "pe_source": ["data/*"],
```